Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_PackageShowView.1.html

```diff
@@ -118,7 +118,7 @@ <h4>When working with a Swift Package Manager manifest:</h4>
                 <span class="branch">main</span>
               </p>
               <form class="copyable_input">
-                <input type="text" data-button-name="Copy Code Snippet" data-event-name="Copy SPM Manifest Code Snippet Button" value=".package(url: &quot;https://github.com/Alamofire/Alamofire&quot;, branch: &quot;main&quot;)" readonly/>
+                <input type="text" data-button-name="Copy Code Snippet" data-event-name="Copy SPM Manifest Code Snippet Button" value=".package(url: &quot;https://github.com/Alamofire/Alamofire.git&quot;, branch: &quot;main&quot;)" readonly/>
```

Comment:
There's just an additional `.git` in all the snippets now, which we were previously not correctly mocking, probably because the release URL we used didn't have it. Either is fine, this is just more realistic, because package urls would have a `.git` suffix.

This is the same for all the updated HTML snapshots.